learn/advanced/pagination.md

```diff
@@ -0,0 +1,130 @@
+# Pagination of search results
```

Comment:
Pagination refers to navigating through results by selecting a page. So it is rather one of many navigation methods.

I would have named the page `Navigating search results` or `Browsing search results`.

That said, it's not a big deal. It might even be interesting to call it pagination so that people can get here more easily because many people use the word pagination to talk about infinite scrolling, load more etc.

You make the call! I just wanted to point out that 😇